examples/translate/translate.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2018-2019 opcua authors. All rights reserved.
+// Use of this source code is governed by a MIT-style license that can be
+// found in the LICENSE file.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+
+	"github.com/gopcua/opcua"
+	"github.com/gopcua/opcua/debug"
+	"github.com/gopcua/opcua/id"
+	"github.com/gopcua/opcua/ua"
+)
+
+func main() {
+	endpoint := flag.String("endpoint", "opc.tcp://localhost:4840", "OPC UA Endpoint URL")
+	flag.BoolVar(&debug.Enable, "debug", false, "enable debug logging")
+	flag.Parse()
+	log.SetFlags(0)
+
+	ctx := context.Background()
+
+	c := opcua.NewClient(*endpoint)
+	if err := c.Connect(ctx); err != nil {
+		log.Fatal(err)
+	}
+	defer c.Close()
+
+	root := c.Node(ua.NewTwoByteNodeID(id.ObjectsFolder))
+	nodeId, err := root.TranslateBrowsePathInSameNamespaceToNodeId(2, "demo_led.temperature")
```

Comment:
I'd say, for the example to be portable, the arguments of `TranslateBrowsePathInSameNamespaceToNodeId()` should be provided as command line flags, just like the endpoint. Maybe add a comment with your values as an example.